tests/output/inp_preprocess.c_asm_output.txt

```diff
@@ -19,6 +20,18 @@ ret
 mov rsp, rbp
 pop rbp
 ret
+global function
+function:
```

Comment:
seems it caused a problem with the `ifdef` keyword. `function` should not be defined twice ([L12](https://github.com/keyvank/30cc/blob/9935bd4987cf59076ecad2080919c47a52c684cc/tests/output/inp_preprocess.c_asm_output.txt#L12), [L24](https://github.com/keyvank/30cc/blob/9935bd4987cf59076ecad2080919c47a52c684cc/tests/output/inp_preprocess.c_asm_output.txt#L24.)).